app/views/admin/claims/_answer_section.html.erb

```diff
@@ -1,7 +1,15 @@
-<h2 class="govuk-heading-m">
+<% description ||= nil %>
+
+<h2 class="govuk-heading-l">
   <%= heading %>
 </h2>
 
+<% if description.present? %>
```

Comment:
We can use the `local_assigns` helper here to avoid having to do the conditional assignment above.

Reply:
Thanks, sorted!